pkg/apps/application-list.jsx

```diff
@@ -98,11 +98,21 @@ export const ApplicationList = ({ metainfo_db, appProgress, appProgressTitle, ac
         comps.push(metainfo_db.components[id]);
     comps.sort((a, b) => a.name.localeCompare(b.name));
 
-    function get_config(name, distro_id, def) {
+    function get_config(name, os_release, def) {
+        // ID is a single value, ID_LIKE is a list
+        const os_list = [os_release.ID || "", ...(os_release.ID_LIKE || "").split(/\s+/)];
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19283-20230906-092432-66756c59-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)